data/src/main/java/org/oppia/data/backends/gae/ModifyJsonInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.*
+
+import java.io.IOException
+
+class ModifyJsonInterceptor : Interceptor {
+
+  @Throws(IOException::class)
+  override fun intercept(chain: Interceptor.Chain): Response {
+    val request = chain.request()
+    val response = chain.proceed(request)
+
+    if (response.code() == 200) {
+      if (response.body() != null) {
+        var rawJson = response.body()!!.string()
+        if (rawJson.startsWith(NetworkSettings.XSSI_PREFIX)) {
+          rawJson = rawJson.substring(rawJson.indexOf('\n') + 1)
+        }
+        val contentType = response.body()!!.contentType()
+        val body = ResponseBody.create(contentType, rawJson)
+        return response.newBuilder().body(body).build()
+      }
+    } else if (response.code() == 403) {
+      //TODO(#5): Identify how error handling will work (e.g. what Retrofit does),
```

Comment:
Per our conversation last week, is there action needed to be taken here? I thought we were just going to rely on the default Retrofit behavior for error handling (e.g. that errors should be propagated to the Retrofit Call objects.

Reply:
Yes, this is not required now and has been removed from the code.